src/browser/components/Tabs/Tabs.scss

```diff
@@ -3,11 +3,11 @@
   flex-wrap: nowrap;
   padding: 0 32px;
   border-bottom: 1px solid #dee2e6;
-
+  width: calc(100vw - 64px);
```

Comment:
@jarindr - is 64px the sidebar width? Looks like a magic number. Was there any reason we didn't just go with 100%?

I'm will work a little on https://github.com/nos/client/issues/187 and might adjust this at the same time.

Reply:
you can either fix the width of the body => to all containers and use 100% width here, but i choose to go with this solution that doesn't need to specify width for all parent containers

Reply:
@jarindr `100vw` is the entire viewport width, the sidebar is 70px so it looks about right in the tabs, but you're 6px off. It only works because the tabs are `box-sizing: border-box`.

I would rather add a `width: calc(100vw - $authenticatedLayout-menu-width)` rule to the main container, and then the children can all rely on 100% width working as expected.

Reply:
@mmoshi you can do that, or you can remove padding and fix the width of tabs and align center it, if you wanna change that there are many ways :) no right or wrong

Reply:
if you really wanna do it right, don't use `vw` but start from the body and the containers inherit the width instead

Reply:
also i think you are right that it was off by 6px didn't check it was `box-sizing: border-box`